css-overflow-5/Overview.bs

```diff
@@ -203,27 +234,104 @@ Overflow Controls: the 'scroll-marker-group' property and pseudo-elements</h2>
 	to ensure that the generation of scroll marker pseudo-elements does not invalidate the layout of the site:
 
 	<pre class="lang-css">
-		/* The generation of ::scroll-marker pseudo-elements cannot invalidate layout outside of this pseudo-element. */
+		/* The generation of ::scroll-marker pseudo-elements shouldn't
+		 * invalidate layout outside of this pseudo-element. */
 		::scroll-marker-group { contain: size !important; }
 	</pre>
 
 	The 'scroll-marker-group' implicitly behaves as a single focusable component,
 	establishing a <a href="https://open-ui.org/components/focusgroup.explainer/">focusgroup</a>.
 
-<h3 id="scroll-marker-pseudo">The ''::scroll-marker'' pseudo-element</h3>
+<h4 id="scroll-marker-pseudo">The ''::scroll-marker'' pseudo-element</h4>
 
```

Comment:
This section jumps straight into the mechanics; could probably do with an intro paragraph. Maybe something like:

"Similar to ::before and ::after, all elements can have a ::scroll-marker pseudo-element, which is collected into the ::scroll-marker-group of the nearest [=scroll container=] ancestor, and scrolls to the element when activated."